src/GaussianCentroid.cc

```diff
@@ -499,7 +498,6 @@ MAKE_TWODG(afw::image::Image<float>);
 MAKE_TWODG(afw::image::Image<double>);
 MAKE_TWODG(afw::image::Image<int>);
 
-} // end anonymous namespace
```

Comment:
I don't think there's any reason to remove the anonymous namespace here - I'm guessing this is a relic from when you had planned to make the {{twodg}} function itself public, rather than a wrapper (I like what you've done with the wrapper much better, but I think that means you can put the anonymous namespace back in).